en/services/component_information.md

```diff
@@ -145,4 +146,4 @@ This can generally be avoided by hosting the files compressed on-vehicle.
 
 We propose that manufacturers that use autopilots with limited flash (1MB or below) and do custom firmware development should host the files in github.
 
-> **Note** File URLs are max 70 chars. If hosted within MAVLink we propose allocated numbers for URLs.  
+> **Note** File URLs are max 100 chars.
```

Comment:
Anything else in this section changed? i.e. do we know how to do translations yet?

Reply:
Don wanted to look into translations.
The limit doesn't apply anymore for external url's, so I'd just remove it.